Keas.Mvc/Controllers/ConfirmController.cs

```diff
@@ -23,20 +23,28 @@ public ConfirmController(ApplicationDbContext context, ISecurityService _securit
         }
 
         public async Task<IActionResult> MyStuff()
-        {
+        {   
             var person = await _securityService.GetPerson(Team);
+            if(person==null){
+                 Message = "You are not yet added to the system.";
+                return RedirectToAction("Index","Home");
```

Comment:
Interesting, we probably want to make this a much nicer experience in the future, maybe a page with info on what to do.